site/package.json

```diff
@@ -91,7 +91,7 @@
   },
   "devDependencies": {
     "@babel/runtime": "^7.17.9",
-    "@playwright/test": "1.23.3",
+    "@playwright/test": "^1.24.0",
```

Comment:
I updated Playwright in hope to avoid the patch, but it was still needed. We got improved clipboard support in Chrome by doing this too. Release notes: https://github.com/microsoft/playwright/releases